tests/django_restframework_gis_tests/views.py

```diff
@@ -132,10 +132,12 @@ class GeojsonLocationNoIdDetails(generics.RetrieveUpdateDestroyAPIView):
 
 
 class LocationFilter(GeoFilterSet):
-    contains_properly = GeometryFilter(name='geometry', lookup_type='contains_properly')
```

Comment:
It is only for the test.
So as long as you do not want test against older django-filter versions this should be fine.